bpaotu/bpaotu/templates/bpaotu/search.html

```diff
@@ -3,7 +3,51 @@
 {% load static from staticfiles %}
 
 {% block extra_head %}
+<script>
+// $(document).ready(function() {
+    var authenticate_users = function() {
+        // !!! NOTE: Do not hardcode the endpoints - so change it eventually.
+        // var bpa_endpoint = window.otu_search_config['ckan_base_url'] + '/user/me';
+        // var bpaotu_endpoint = window.otu_search_config['ckan_base_url'] + '/bpa/otu/otu_verification_endpoint';
+        var bpa_endpoint = 'https://localhost:8443/user/check_permissions';
+        var bpaotu_endpoint = 'https://localhost:8443/bpa/otu/otu_verification_endpoint';
+
```

Comment:
you can't hardcode these URLs, use relative URLs

Reply:
Fixed.